src/components/EmailNotifications/EmailNotificationsManager.js

```diff
@@ -651,35 +651,60 @@ const EmailNotificationsManager = React.memo(
   }
 )
 
-const WrappedContainer = React.memo(function AnimatedModal({ ...props }) {
+const WrappedContainer = React.memo(function WrappedContainer({ ...props }) {
   const [insideModal] = useInside('NotificationsModal')
   const { screenId } = props
 
-  if (insideModal || screenId === NOTIFICATIONS_PREFERENCES_SCREEN) {
-    return <AnimatedContainer {...props} />
-  }
+  const Container =
+    insideModal || screenId === NOTIFICATIONS_PREFERENCES_SCREEN
+      ? AnimatedContainer
+      : AnimatedBox
+
+  return <Container {...props} />
+})
+
+const AnimatedBox = ({ ...props }) => {
+  const ref = useRef(null)
+  const [height, setHeight] = useState(0)
+
+  const setRef = useCallback(node => {
+    if (node) {
+      setHeight(node.clientHeight)
+    }
+
+    ref.current = node
+  }, [])
 
   return (
-    <Box
-      css={`
-        display: flex;
-        justify-content: center;
-      `}
-    >
+    <Box padding={5 * GU}>
       <div
         css={`
+          position: relative;
           max-width: 800px;
+          margin: 0 auto;
         `}
       >
-        <AnimatedContainer {...props} />
+        <Spring
+          config={springs.smooth}
+          from={{ height: `${38 * GU}px` }}
+          to={{ height: `${height}px` }}
+          native
+        >
+          {({ height }) => (
+            <animated.div style={{ height }}>
+              <AnimatedContainer {...props} refCallback={setRef} />
+            </animated.div>
+          )}
+        </Spring>
       </div>
     </Box>
   )
-})
+}
 
 const AnimatedContainer = React.memo(function AnimatedModal({
   screenId,
   children,
+  refCallback,
```

Comment:
We may want to use [`forwardRef()`](https://reactjs.org/docs/forwarding-refs.html) instead.